controllers/cloud.redhat.com/providers/kafka/strimzi.go

```diff
@@ -5,13 +5,14 @@ import (
 	"strconv"
 
 	crd "cloud.redhat.com/clowder/v2/apis/cloud.redhat.com/v1alpha1"
-	strimzi "cloud.redhat.com/clowder/v2/apis/kafka.strimzi.io/v1beta1"
+	strimzi "github.com/bsquizz/strimzi-client-go/apis/kafka.strimzi.io/v1beta1"
```

Comment:
can we have this moved to the redhatinsights org?

Reply:
Done